src/rpc/channel.rs

```diff
@@ -290,6 +291,10 @@ where
                     funding_amount: params.funding_amount,
                     public: params.public.unwrap_or(false),
                     shutdown_script: params.shutdown_script.clone().map(|s| s.into()),
+                    commitment_delay_epoch: params
```

Comment:
- The delay epoch is not validated here. It can be too long or too short.
- There are some bit operation involved in `EpochNumberWithFraction`. We are accepting a u64 as parameter here, but it can be hard for human to construct a valid and meaningful u64 for this `EpochNumberWithFraction` here.

Reply:
resolved in commit 088e0db1ca1f598d73d89fd0d3975f773412cdee